### ISSUE TITLE
Grid not displaying any pixels

### ISSUE THREAD

**Issue by tomreitsma** [dpren/react-pixel-grid#1]
Hi there! It seems the grid isn't actually displaying any content. The background and grid size are being rendered just fine, but the actual cells don't seem to be there.

package.json:

```json
{
  "name": "pixel-grid",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "@testing-library/jest-dom": "^4.2.4",
    "@testing-library/react": "^9.3.2",
    "@testing-library/user-event": "^7.1.2",
    "pixel-grid": "^1.0.2",
    "react": "^16.13.1",
    "react-dom": "^16.13.1",
    "react-pixel-grid": "^0.1.1",
    "react-scripts": "3.4.3"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
  "eslintConfig": {
    "extends": "react-app"
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  },
  "devDependencies": {}
}
```

**Comment by tomreitsma** [comment]
Update: It seems to have something to do with the render needing to be called again after being loaded. The initial demo doesn't work because there's only one frame being rendered. 

If there's more it seems to work fine. Here's my App.js I used for this demo:

```javascript
import React, {Component} from "react";
import PixelGrid from "react-pixel-grid";

class App extends Component {
  constructor() {
    super();
    const matrix = base => Array(Math.pow(base, 2)).fill('#CCC');

    let grid_size = 100,
      data = matrix(grid_size);

    this.state = {
      size: 10,
      grid_size: grid_size,
      data: data,
      padding: 1,
      resolution: 100,
      fps: 60,
      interval: null
    }
  }

  componentDidMount() {
    const {fps} = this.state;
    let interval = setInterval(() => this.timer(), 1000 / (fps || 0.00001));
    this.setState({interval: interval});
  }

  componentWillUnmount() {
    clearInterval(this.state.interval);
  }

  timer() {
    const {grid_size} = this.state;
    let new_data = this.state.data;

    let x = Math.floor(Math.random() * (grid_size * grid_size))

    new_data[x] = '#' + Math.floor(Math.random() * (999 - 100 ));

    this.setState({
      data: new_data
    });
  }

  render() {
    const {size, data, padding, resolution, fps} = this.state;

    return (
      <div>
        px size: <input type="number" value={size} min={0} max={100} onChange={ev => this.setState({size:Math.min(100, Number(ev.currentTarget.value))})} />
        padding: <input type="number" value={padding} min={0} max={20} onChange={ev => this.setState({padding:Math.min(20, Number(ev.currentTarget.value))})} />
        resolution: <input type="number" value={resolution} min={1} max={600} onChange={ev => this.setState({resolution:Math.min(600, Number(ev.currentTarget.value))})} />
        fps: <input type="number" value={fps} min={1} max={100} onChange={ev => this.setState({fps:Math.min(80, Number(ev.currentTarget.value))})} />

        <PixelGrid
          data={data}
          options={{
            size,
            padding,
            background: [1, 1, 1],
          }}
        />

      </div>
    );
  }
}

export default App;
```



<img width="544" alt="Screenshot 2020-09-20 at 14 56 44" src="https://user-images.githubusercontent.com/17272541/93711812-8164a180-fb51-11ea-9499-34c488c4f59c.png">

**Comment by dpren** [comment]
@tomreitsma Thanks for the report. The initial render was indeed getting mis-scheduled, but I'm glad you figured it out. Patched with 0.1.2
